src/styles.css

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+p {
+  margin: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+body {
+  font-weight: 400;
+  font-style: normal;
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-left: 0;
+
+  list-style: none;
+  border-radius: var(--border-radius);
+  border: 1px solid #f3f3f3;
+
+  --card-margin: 0 16px 16px;
+  --border-radius: 5px;
+}
+
+.card__img {
+  margin: 32px 19px 40px;
+}
+
+.card__info,
+.card__rating,
+.card__price {
+  margin: var(--card-margin);
+}
+
+.card__info p:first-child {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin: 0 0 4px;
+}
+
+.card__info p:last-child {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__rating {
+  margin-bottom: 23px;
+}
+
+.card__rating,
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__review {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.card__price p:first-child {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price p:last-child {
+  font-size: 16px;
+  font-weight: 700;
+}
```

Comment:
Add classes to this paragraphs as Jamal advised and change selectors in lines 97 and 102 to only classes. For example `card__price-tag`